schema.sql

```diff
@@ -9,5 +9,6 @@ CREATE TABLE animals (
     escape_attempts integer NOT NULL,
     neutered boolean NOT NULL,
     weight_kg decimal NOT NULL,
+    species text,
```

Comment:
**Changes Required**
- [ ] We shouldn't add the `species` column directly to the `animal` table but rather use the `ALTER` command to do so. Let us try and make these changes.